src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,48 @@
+package duke;
+
+import java.io.FileNotFoundException;
+import java.util.List;
+import java.util.ArrayList;
+
+/**
+ * Represents the outward class used by users to interact with Duke application.
+ */
+public class Duke {
+
+    private final Storage storage;
+    private TaskList tasks;
+    private final Ui ui;
+
+    public Duke(String filepath) {
+        ui = new Ui();
+        storage = new Storage(filepath);
+        try {
+            tasks = storage.load();
+        } catch (FileNotFoundException err) {
+            ui.showLoadingError();
+            tasks = new TaskList();
+        }
+    }
+
+    public static void main(String[] args) {
+        new Duke("data/tasks.txt").run();
+    }
+
+    private void run() {
+        ui.intro();
+        //int id = 0;
```

Comment:
I think you forgot to remove this commented-out piece of code!

Reply:
Thanks for all the pointers!